src/kani-compiler/src/codegen_cprover_gotoc/overrides/hooks.rs

```diff
@@ -98,13 +99,19 @@ impl<'tcx> GotocHook<'tcx> for ExpectFail {
         assert_eq!(fargs.len(), 2);
         let target = target.unwrap();
         let cond = fargs.remove(0).cast_to(Type::bool());
-        // add "EXPECTED FAIL" to the message because compiletest relies on it
+
+        // msg is the comment field that's set on source location which is presented as the Description field
```

Comment:
I'd just remove the comments in this function. I don't think they're adding anything?

But you removed a comment that maybe still is: we're adding "EXPECTED FAIL" because of compiletest. We shouldn't lose that comment until we fix compiletest (or our own output formatter?) to handle the `expect_fail` property class correctly.

Reply:
cool, i'll add it back